### DISCUSSION TITLE
Creation of Limitations Section

### DISCUSSION THREAD

**Discussion by ralph-dev** [LonelyCpp/react-native-youtube-iframe#93] [General]
The docs under Basic Usage should have a limitations section similar style to this https://reactnavigation.org/docs/limitations

I've used this library as well as many others react-native-video etc and found that there is a significant limitation with being able to play certain copyrighted videos on Mobile applications (Specifically React Native).

**Limitations**
_While this library does allow you to play youtube videos in your mobile app, keep in mind if the youtube video is copyrighted or embed mode is disabled then it will not render and you will get video unavailable. A workaround is described here that allows you to play ANY youtube video on your mobile app but requires a bit more work._

Helpful Links:
**See Method 4**
https://surajmdurgad.medium.com/embedding-youtube-videos-in-a-react-native-app-8c556a18fd76

I've also tested these libraries, they don't work either, only the Iframe of an Iframe inside a Website works.

- react-native-video
- react-native-youtube-iframe (This Lib)
- react-native-youtube

This will help save developers and maintainers time if the library they are using explicitly defines its limitations / scope. Let me know your thoughts!

**Comment by LonelyCpp** [comment]
Hey wow, sorry for the late response! I totally forgot about the new discussions tab.

I Think it's definitely a good idea to add this as a PSA, and perhaps list some workarounds like 
* https://github.com/LonelyCpp/react-native-youtube-iframe/issues/89
* link to this blog post (here this library won't be of use lol)

**Reply by ralph-dev** [reply]
@LonelyCpp Can you create the PSA I'm not sure what's happening in #89 . I am no longer using this repo but glad to have made some contributions! If that gets fixed though I'd love to read the PSA and see if it works / test it out!

**Comment by LonelyCpp** [comment]
This has been fixed with `2.0.0` :)

The iframe code defaults to a hosted page on https://lonelycpp.github.io/react-native-youtube-iframe/iframe.html

Since this overcomes the empty base URL issue - rogue `embed_not_allowed` errors is not an issue!
